eng/pipelines/templates/stages/vmr-validation.yml

```diff
@@ -65,7 +65,7 @@ stages:
   jobs:
   - job: ValidateInstallers_Linux_x64
     pool: ${{ parameters.pool_Linux }}
-    timeoutInMinutes: 30
+    timeoutInMinutes: 60
```

Comment:
Curious why this needed to be increased?

Reply:
It failed once due to download of required artifacts that took longer than usual - [build](https://dev.azure.com/dnceng/internal/_build/results?buildId=2785488&view=logs&s=93594280-c135-5b37-42ba-1687921e6d94&j=6e593626-f6df-5699-84b9-2127a964bf7b)